doc/api/vm.md

```diff
@@ -1169,6 +1218,26 @@ the `process.nextTick()` and `queueMicrotask()` functions.
 This issue occurs because all contexts share the same microtask and nextTick
 queues.
 
+## `vm.constants`
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} An object containing commonly used constants for the vm module.
+
+### `vm.constants.measureMemory`
```

Comment:
Could this perhaps just be replaced with strings of `'summary'`, `'detailed'` which we will then convert to appropriate constants upon call? I think that will be simpler for the end-user without introducing too much trouble on our end.

Reply:
I'd definitely prefer just the simple string versions to avoid the use of the constants.

Reply:
hm, I guess using string constants do make more sense here, as with exported constants even if one store them in a variable first they'd still need to type it every time they make the call, which is probably a pair of quotes less compared to string constants.